### ISSUE TITLE
Puml development and caching

### ISSUE THREAD

**Issue by jtreher** [plantuml/plantuml#227]
It appears as though the StringUtils.DiagramIsCachable() is checking for certain words following StartUml. Otherwise, it is emitting the cache headers. Whenever I am doing local work on puml files with the PlantUML-Server, I am required to manually bust cache for the image (and figuring out what was happening was no easy task either). Natrually, I believe this is because the puml contents are not considered in the encoded text.

Should there be an environment setting which overrides this similar to the ALLOW_PLANTUML_INCLUDE or maybe have that setting automatically trigger?

**Comment by arnaudroques** [comment]
Just to be sure:
You'd like that we create a environment variable `PLANTUML_CACHE_HTTP_HEADER` so that when this variable is set to `false`, we do not emit any cache directive in http answer here:

https://github.com/plantuml/plantuml-server/blob/master/src/main/java/net/sourceforge/plantuml/servlet/DiagramResponse.java#L99

Did we get the idea ?

**Comment by jtreher** [comment]
@arnaudroques Yes, that is correct. I had started working on this but I could not get the project to build because it would not pass the style checker.

**Comment by jtreher** [comment]
@arnaudroques Should I move this issue over to PlantUML-server rather than here?
